types/bbcode-to-react/index.d.ts

```diff
@@ -185,11 +159,42 @@ declare class Parser<T = { linkify: boolean }> {
      */
     toReact(input: string): ReactNode;
 }
+declare namespace bbcodeToReact {
+    /**
+     * An interface for class member in "Tag"
+     */
+    interface TagType {
+        /**
+         * Tag name, with type string
+         */
+        name: string;
+        /**
+         * The parent of a tag object, with type ReactNode
+         */
+        parent: ReactNode;
+        /**
+         * The inner text of a tag object, with type string
+         */
+        text: string;
+        /**
+         * A "params" object for inner processing
+         */
+        params: object;
+        /**
+         * The children of a tag object, with type ReactNode array
+         */
+        children: ReactNode[];
+    }
+
+    type bbcodeToReactExport = Parser & {
```

Comment:
This type will be exported implicitly; that seems undesirable. Does this type really have to be declared in the namespace here?

Reply:
Done.